tests/test_fixtures/configuration_for_testing_v2_v3_migration/spark/v2/great_expectations/expectations/Titanic/profiled.json

```diff
@@ -0,0 +1,47 @@
+{
+  "data_asset_type": null,
+  "expectation_suite_name": "Titanic.profiled",
+  "expectations": [
+    {
+      "expectation_type": "expect_table_columns_to_match_ordered_list",
+      "ge_cloud_id": null,
+      "kwargs": {
+        "column_list": [
+          "_c0",
+          "Name",
+          "PClass",
+          "Age",
+          "Sex",
+          "Survived",
+          "SexCode"
+        ]
+      },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_table_row_count_to_be_between",
+      "ge_cloud_id": null,
+      "kwargs": {
+        "max_value": 1313,
+        "min_value": 1313
+      },
+      "meta": {}
+    }
+  ],
+  "ge_cloud_id": null,
```

Comment:
Are we confident that `ge_cloud_id` shows up with the latest version of GE? I did some work while on DevRel to remove null ids but I forget where that landed.

Don't think it's an issue to keep just curious!

Reply:
Yes, just checked and the `ge_cloud_id` value does show up with the latest version :)